core/src/wallet/tezos/explorers/ExternalTezosLikeBlockchainExplorer.cpp

```diff
@@ -154,27 +172,31 @@ namespace ledger {
                         // OR
                         // tzindex v12+ gas_price compute
                         else {
-                            if (!json.HasMember(gasLimitField) || !json[gasLimitField].IsString()) {
+                            if (!json.HasMember(gasUsedField) || !json[gasUsedField].IsString()) {
                                 throw make_exception(
                                     api::ErrorCode::HTTP_ERROR, fmt::format(
                                         "Failed to compute gas_price from network, no (or malformed) field \"{}\" in response",
-                                        gasLimitField));
+                                        gasUsedField));
                             }
                             if (!json.HasMember(feeField) || !json[feeField].IsString()) {
                                 throw make_exception(
                                     api::ErrorCode::HTTP_ERROR, fmt::format(
                                         "Failed to compute gas_price from network, no (or malformed) field \"{}\" in response",
                                         feeField));
                             }
-                            const uint64_t apiGasLimit = std::stoull(json[gasLimitField].GetString());
-                            if (apiGasLimit == 0) {
+                            const uint64_t apiGasUsed = std::stoull(json[gasUsedField].GetString());
+                            if (apiGasUsed == 0) {
                                 throw make_exception(
-                                    api::ErrorCode::HTTP_ERROR, "Failed to compute gas_price from network, gas_limit of HEAD block is 0"
+                                    api::ErrorCode::HTTP_ERROR, "Failed to compute gas_price from network, gas_used of HEAD block is 0"
                                 );
                             }
-                            const double apiFee = std::stod(json[feeField].GetString());
-                            const double numericGasPrice = apiFee / static_cast<double>(apiGasLimit);
-                            gasPrice = std::to_string(numericGasPrice);
+
+                            double apiFee = std::stod(json[feeField].GetString());
```

Comment:
`std::stod` could potentially throw an exception if the value of `feeField` cannot be converted. Perhaps should we `try/catch` it?

Reply:
(Same thing for `std::stoull` above).

Reply:
fyi+: LL is not longer following our changes. So don't worry @ktollec-ledger, that's not to be supported.

Reply:
Fixed in 0d37eacba35870aadd1d883088d1c7b2a3e1b8c6